src/uu/pidwait/src/pidwait.rs

```diff
@@ -42,11 +43,23 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
         }
     }
 
-    wait(&proc_infos);
+    // It should be fine to reserve a `timeout` parameter for future use.
+    wait(&proc_infos, None)?;
 
     Ok(())
 }
 
+pub(crate) fn wait(
+    procs: &[ProcessInformation],
+    timeout: Option<Duration>,
+) -> Result<Option<()>, std::io::Error> {
+    if !procs.is_empty() {
```

Comment:
A detail: I would drop the `!` and swap the `if` and `else` blocks, it's a little bit easier to read.